### DISCUSSION TITLE
qustion for single_test

### DISCUSSION THREAD

**Discussion by anlei-code** [NCAR/noahmp#151] [Q&A] [answered]
I use the docker to install the Noah-mp,the compliing-process had succeed.But when I test single_point  ,the error have appeared: Problem opening netcdf file: '../HRLDAS_forcing/run/examples/single_point/199801010000.LDASIN_DOMAIN1',please help me.

**Comment by cenlinhe** [comment, marked as answer]
Did you successfully generate the forcing file "../HRLDAS_forcing/run/examples/single_point/199801010000.LDASIN_DOMAIN1"? Maybe double check the data path/folder where you generated those forcing files to make sure you have the correct data path in the namelist.hrldas for those input files.

**Reply by anlei-code** [reply]
Thank you for your reply,this problem has been sloved.
